Tests/test_content.py

```diff
@@ -486,24 +599,85 @@ def run_test_scenario(t, proxy, default_test_timeout, skipped_tests_conf, nightl
 
     run_test(demisto_api_key, proxy, failed_playbooks, integrations, unmockable_integrations, playbook_id,
              succeed_playbooks, test_message, test_options, slack, circle_ci,
-             build_number, server, build_name, is_ami)
-
-
-def execute_testing(server, server_ip, server_version, server_numeric_version, is_ami=True):
-    print("Executing tests with the server {} - and the server ip {}".format(server, server_ip))
-
-    options = options_handler()
-    conf_path = options.conf
-    secret_conf_path = options.secret
-    is_nightly = options.nightly
-    is_memory_check = options.memCheck
-    slack = options.slack
-    circle_ci = options.circleci
-    build_number = options.buildNumber
-    build_name = options.buildName
-
-    conf, secret_conf = load_conf_files(conf_path, secret_conf_path)
-    demisto_api_key = secret_conf.get('temp_apikey')
+             build_number, server, build_name, is_ami, thread_index=thread_index, prints_manager=prints_manager)
+
+
+def restart_demisto_service(ami, demisto_api_key, server, thread_index=0, prints_manager=None):
+    client = demisto_client.configure(base_url=server, api_key=demisto_api_key, verify_ssl=False)
+    ami.check_call(['sudo', 'service', 'demisto', 'restart'])
+    exit_code = 1
+    for _ in range(0, SERVICE_RESTART_TIMEOUT, SERVICE_RESTART_POLLING_INTERVAL):
+        sleep(SERVICE_RESTART_POLLING_INTERVAL)
+        if exit_code != 0:
+            exit_code = ami.call(['/usr/sbin/service', 'demisto', 'status', '--lines', '0'])
+        if exit_code == 0:
+            check_login_message = "{}: Checking login to the server... ".format(datetime.now())
+            prints_manager.add_print_job(check_login_message, print, thread_index)
+            try:
+                res = demisto_client.generic_request_func(self=client, path='/health', method='GET')
+                if int(res[1]) == 200:
+                    return
+                else:
+                    failed_verify_login_message = "Failed verifying login (will retry). status: {}. " \
+                                                  "text: {}".format(res.status_code, res.text)
+                    prints_manager.add_print_job(failed_verify_login_message, print, thread_index)
+            except Exception as ex:
+                error_message = "Failed verifying server start via login: {}".format(ex)
+                prints_manager.add_print_job(error_message, print_error, thread_index)
+
+    raise Exception('Timeout waiting for demisto service to restart')
```

Comment:
i think that @avidan-H removed this

Reply:
yea, I don't think restarting demisto service is necessary. It stam eats up time. Can make an API call to demisto client to reset containers instead.